src/components/Slider/KeenSlider.vue

```diff
@@ -48,6 +48,7 @@ enum SliderPlayingStates {
 
 interface Props {
 	interval?: number;
+	delay?: number;
```

Comment:
You could be more specific about what's being delayed, e.g. `startDelay`. But I can live with `delay`